airflow/bin/cli.py

```diff
@@ -491,16 +491,19 @@ def worker(args):
     from airflow.executors.celery_executor import app as celery_app
     from celery.bin import worker
 
+    concurrency = args.concurrency or conf.get('celery', 'celeryd_concurrency')
```

Comment:
Actually, I think it just stores `'celeryd_concurrency'` as the value for concurrency, which I am not sure we want.